docs/layering.md

```diff
@@ -27,16 +27,13 @@ For the purpose of this tutorial we are using QEMU, but you can use a cloud envi
 ```bash
 # This is the 'next' image we will be using.
 # Anything newer than this should work too.
-$ RELEASE="35.20211215.1.0"
+$ RELEASE_URL=$(curl https://builds.coreos.fedoraproject.org/streams/next.json | jq -r '.architectures.x86_64.artifacts.qemu.formats."qcow2.xz".disk.location')
```

Comment:
Aside: the canonical way to download images is `coreos-installer download`. It will verify the image and decompress it for you.